src/bundle/DependencyInjection/Compiler/RichTextHtml5ConverterPass.php

```diff
@@ -24,13 +24,13 @@ public function process(ContainerBuilder $container)
     {
         if ($container->hasDefinition('ezrichtext.converter.output.xhtml5')) {
             $html5OutputConverterDefinition = $container->getDefinition('ezrichtext.converter.output.xhtml5');
-            $taggedOutputServiceIds = $container->findTaggedServiceIds('ezrichtext.converter.output.xhtml5');
+            $taggedOutputServiceIds = $container->findTaggedServiceIds('ezplatform.ezrichtext.converter.output.xhtml5');
```

Comment:
Should we introduce BC layer for the old `ezrichtext.converter.output.xhtml5` tag? I mean something similar to what we did in case of field type related tags? https://github.com/ezsystems/ezpublish-kernel/blob/94214dfc4c6d3af87714e1eb3b800972f7b7a18e/eZ/Publish/Core/Base/Container/Compiler/AbstractFieldTypeBasedPass.php#L36-L46

Reply:
> Should we introduce BC layer for the old `ezrichtext.converter.output.xhtml5` tag? I mean something similar to what we did in case of field type related tags? https://github.com/ezsystems/ezpublish-kernel/blob/94214dfc4c6d3af87714e1eb3b800972f7b7a18e/eZ/Publish/Core/Base/Container/Compiler/AbstractFieldTypeBasedPass.php#L36-L46

I wondered the same myself. This service tag was named like that since eZ Platform 2.4, so per OSS BC best practices we should. However we've decided that for 3.0 we don't want code polluted with deprecation.
Let me suggest a compromise. Maybe I'll re-target this to `1.1` (corresponding to the next eZ Platform `2.5.x LTS`) and add it there in BC safe way with deprecation? Then, for `master` I would drop old service tag.